internal/pkg/override/errors.go

```diff
@@ -17,3 +19,12 @@ func (err *errNPMUnavailable) Error() string {
 func (err *errNPMUnavailable) RecommendActions() string {
 	return fmt.Sprintf(`Please follow instructions at: %q to install "npm"`, "https://docs.npmjs.com/downloading-and-installing-node-js-and-npm")
 }
+
+// ErrNotExist occurs when an overrides/ directory is either empty or does not exist.
+type ErrNotExist struct {
```

Comment:
How about renaming this as `ErrDirNotExist` for clearer understanding of what does not exist.